notebook/notebookapp.py

```diff
@@ -1820,10 +1854,17 @@ def launch_browser(self):
         else:
             open_file = self.browser_open_file
 
-        b = lambda: browser.open(
-            urljoin('file:', pathname2url(open_file)),
-            new=self.webbrowser_open_new)
-        threading.Thread(target=b).start()
+        if not self.use_redirect_file:
```

Comment:
If practical, can we have the if/else the other way round, so it doesn't need `if not`?